internal/shorthand/env_var.go

```diff
@@ -32,3 +53,7 @@ func MapEnvToEnvConfig(input []shared.DeploymentV3Env) []shared.DeploymentConfig
 
 	return output
 }
+
+func ErrInvalidEnvVarFormat(env string) error {
+	return fmt.Errorf("invalid env var format: %s", env)
+}
```

Comment:
nit I generally prefer for exported members that start with `Err` to be passable into `errors.Is` or `errors.As`, maybe `NewErrInvalidEnvVarFormat`?